src/github.com/getlantern/lantern-ui/app/partials/error.html

```diff
@@ -0,0 +1,46 @@
+<div id="error" ng-controller="ErrorCtrl" ng-show="showError">
+  <div id="oops">
+    {{ 'ERROR_OOPS' | translate }}
+  </div>
+  <div id="error-description">
+    {{ 'ERROR_DESCRIPTION' | translate }}
+  </div>
+  <div id="error-reasons">
+    <ol>
+      <li ng-show="!model.settings.systemProxy">
```

Comment:
I don't think we want this @fffw, as this is a very common issue with the system proxy on or off -- i.e. some issue with extensions right?